src/io-methods/google-cloud-storage.js

```diff
@@ -2,10 +2,14 @@ const { Storage } = require("@google-cloud/storage");
 
 const storage = new Storage();
 
-const regex = /^google:\/\/(.+)\/(.+)$/;
+const regex = /^(?:google|gs):\/\/(.+)\/(.+)$/;
```

Comment:
I'm open to this change, as `gs` is the more common url pattern for google storage. Could you update the documentation in the readme to use `gs://` instead of `google://`?

Reply:
https://github.com/single-spa/import-map-deployer/pull/93/commits/06314a2730b4f537a8ed40d6e6836e0f6ace7c16 done